specification/subscription/resource-manager/Microsoft.Subscription/preview/2023-05-01-preview/subscriptions.json

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
You have a lot of paths missing from previous stable version (2021-10-01: [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json#L19:5](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json#L19:5)). That is also causing Breaking changes in your PR, which if desired to be removed, need approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com>.

Reply:
This is still not addressed. Please check above.